src/Models/RedisDistributedCacheEntry.cs

```diff
@@ -0,0 +1,38 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="RedisDistributedCacheEntry.cs" company="GlobalLink Vasont">
+// Copyright (c) GlobalLink Vasont. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+namespace Vasont.AspnetCore.ServiceStackRedis.Models
+{
+    using System;
+
+    /// <summary>
+    /// This class is used for representing cache item
+    /// </summary>
+    public class RedisDistributedCacheEntry<T> : RedisDistributedCacheEntry
+    {
+        /// <summary>
+        /// Gets or sets value
+        /// </summary>
+        public T Value { get; set; }
+    }
+
+    public class RedisDistributedCacheEntry
+    {
+        /// <summary>
+        /// Gets or sets sliding expiration value
+        /// </summary>
+        public TimeSpan? SlidingExpiration { get; set; }
+
+        /// <summary>
+        /// Gets or sets absolute expiration time
+        /// </summary>
+        public DateTimeOffset? AbsoluteExpiration { get; set; }
+
+        /// <summary>
```

Comment:
Why do we care about last accessed? In a sliding expiration scenario, the EXPIRE time just keeps getting reset on each access get/set. That means, the client would set Expires command on each Get/Set.  In an absolute scenario, the Expires sets only on the Set command. The timeout is always the same however in seconds.

Reply:
You are right - removed this field